docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>

```

Comment:
<a name="textField4"></a> this is the anchor tag ### TextField Properties so it is better to create something unique here.
change here to:

<a name="inputfield"></a>

Reply:
Okay, Changed to <a name="input-properties"></a>